cartographer_grpc/framework/server.cc

```diff
@@ -0,0 +1,47 @@
+#include "cartographer_grpc/framework/server.h"
+
+#include "glog/logging.h"
+
+namespace cartographer_grpc {
+namespace framework {
+
+void Server::Builder::SetNumberOfThreads(size_t number_of_threads) {
+  options_.number_of_threads = number_of_threads;
+}
+
+void Server::Builder::SetServerAddress(const std::string& server_address) {
+  options_.server_address = server_address;
+}
+
+std::unique_ptr<Server> Server::Builder::Build() {
+  std::unique_ptr<Server> server(new Server(options_));
```

Comment:
`make_unique`?

Reply:
I can't use make_unique here as the constructor for Server is private and calling it from make_unique fails.